client/src/screens/groups/components/styles.js

```diff
@@ -0,0 +1,281 @@
+import { StyleSheet } from 'react-native';
+
+const PADDING = 8;
+const BORDER_RADIUS = 5;
+const FONT_SIZE = 16;
+const HIGHLIGHT_COLOR = 'rgba(0,118,255,0.9)';
+
+const styles = StyleSheet.create({
```

Comment:
there's a lot of stuff in here.. is all of this actually used?

Reply:
no. a lot of it isnt